hedera/urls.py

```diff
@@ -34,6 +36,10 @@
     path("api/v1/personal_vocab_list/", api.PersonalVocabularyListAPI.as_view()),
     path("api/v1/personal_vocab_list/<int:pk>/", api.PersonalVocabularyListAPI.as_view()),
 
+    path("lti/", include("lti_provider.urls")),
+    path("lti/lti_initializer/", LtiInitializerView.as_view(), name="lti_initializer"),
+    path("lti/lti_registration", LtiRegistrationView.as_view(), name="lti_registration"),
```

Comment:
Can we add the trailing slash for consistency with other urls?

```suggestion
    path("lti/lti_registration/", LtiRegistrationView.as_view(), name="lti_registration"),
```

Reply:
Sure thing, I'll changed that up.